evergreen/config.yml

```diff
@@ -774,7 +774,7 @@ task_groups:
   - long-running-core-tests
 
 - name: fuzzer-tests
-  setup_group_can_fail_task: true
+  setup_group_can_fail_task: false
```

Comment:
So what this will do is make it so that if fetching the source/binaries/compiling fails that it won't actually fail the whole task group and will still try to run the fuzzer even if it failed to compile. Is that what you want?

Reply:
No, I want to avoid to fail the build if the fuzzer does not find anything. If there is no crash after 60min.